### DISCUSSION TITLE
Live Stream opens automatically after ding on Fire Tablet

### DISCUSSION THREAD

**Discussion by laumar01** [tsightler/ring-mqtt#531] [Support Q&A]
At first thank you very much for this great add-on! I'm pretty new in the ha-community and not a programmer.
At the moment I'm bulding a dashboard to control lights, shutters and other smart things at my home. As control panel I'm using a Kindle Fire HD10 Tablet, which is positioned at our hallway. I installed fully kiosk browser, so that the dashboard is always on the screen.  With your add-on I succesfully put the live stream on the dashboard.
Now I want to build an automation which works the following way:
If a person dings the ring the tablet monitor should switch on and the live stream should automatically open in foreground. 

I'm sure that this is very easy, but as I don't have the home assistant and coding experience, i hope that you are able to help me.

**Comment by tsightler** [comment]
I would guess that you'd have more success asking such a question on the Home Assistant forums vs here as there's nothing really unique about the Ring cameras in this regard and I'm only a very light Home Assistant user.
